projects/packages/forms/src/blocks/input-range/index.js

```diff
@@ -0,0 +1,45 @@
+import { __ } from '@wordpress/i18n';
+import defaultSettings from '../shared/settings';
+import edit from './edit';
+import SliderIcon from './icon';
+import save from './save';
+
+const name = 'input-range';
+const settings = {
+	...defaultSettings,
+	title: __( 'Range input', 'jetpack-forms' ),
+	description: __( 'Collect a value from site visitors using a slider.', 'jetpack-forms' ),
+	icon: {
+		src: <SliderIcon />,
+	},
+	parent: [ 'jetpack/field-slider' ],
+	attributes: {
+		...defaultSettings.attributes,
+		min: {
+			type: 'number',
+			default: 0,
+		},
+		max: {
+			type: 'number',
+			default: 100,
+		},
+		default: {
+			type: 'number',
+			default: 50,
```

Comment:
Wondering it would make sense to not be opinionated about default being right in the middle, and default this also to 0?

Reply:
Updated this to 0.